src/components/common/CommonItem/index.tsx

```diff
@@ -71,12 +86,13 @@ function CommonItem({
           )}
           <View style={styles.leftContainer}>
             <Text style={FontStyles.Normal}>{name || id}</Text>
-            <Text style={FontStyles.NormalGray}>{formattedId}</Text>
+            <Text style={FontStyles.NormalGray}>{subtitle || formattedId}</Text>
```

Comment:
maybe we could remove formattedId and send it always as subtitle 🤔

Reply:
I think this way works pretty well, IMHO